controls/radchartview3d/features/palettes.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Palettes
+page_title: Palettes
+description: Palettes
+slug: radchartview3d-features-palettes
+tags: 3d,chartview,cartesianchart3d,palette,materialgroup,material,color,brush
+published: True
+position: 1
+---
+
+# Palettes
+
+__RadChartView3D__ has a built-in automatic coloring mechanism. There are predefined palettes containing 3D materials used to define the textures of the chart data points.
+
+#### __Figure 1: List with the predefined chart palettes__
+![](images/radchartview-3d-features-palettes-0.png)
+
+A palette can be set via the __Palette__ property of the chart. The property is of type __System.Windows.Media.Media3D.MaterialCollection__ so you can easily create a [custom palette](#create-a-custom-palette).
+
+#### __[XAML] Example 1: Setting a palette in XAML__
+{{region xaml-radchartview3d-features-palettes-0}}
+	<telerik:RadCartesianChart3D Palette="Windows8" />
+{{endregion}}
+
+You can access the predefined palettes in code using the __Chart3DPalettes__ static class.
+
+#### __[C#] Example 2: Setting a palette in code__
+{{region cs-radchartview3d-features-palettes-1}}
+	chart.Palette = Chart3DPalettes.Windows8;
+{{endregion}}
+
+Each series added in the chart will use different entry from the MaterialCollection (the palette). If you have a single series only the first material from the collection will be used.
```

Comment:
If you have a single series, only the first material from the collection is used.